cargo-pgrx/src/command/install.rs

```diff
@@ -395,8 +401,16 @@ pub(crate) fn find_library_file(
     Ok(library_file_path)
 }
 
+static CARGO_VERSION: Lazy<MemoizeKeyValue> = Lazy::new(|| Arc::new(Mutex::new(HashMap::new())));
+
 pub(crate) fn get_version(manifest_path: impl AsRef<Path>) -> eyre::Result<String> {
-    match get_property(&manifest_path, "default_version")? {
+    let path_string = manifest_path.as_ref().display().to_string();
```

Comment:
Not really the best way to do this but we do it everywhere in this code so it's fine.

Reply:
Should I use `PathBuf`?

Reply:
~~I think its fine. I wouldn't have approved it if I thought it needed to change. We do this all over cargo-pgrx anyway.~~

Well, you did it anyway while I wrote this, so it's hard to complain about it.

Reply:
That's great, thank you!